.pre-commit-config.yaml

```diff
@@ -10,6 +23,7 @@ repos:
   - id: pylint
     name: pylint
     entry: pylint
+    args: [--extension-pkg-whitelist=pygit2]
```

Comment:
Can this be added to the pylint config in `pyproject.toml` instead? I believe it would go under `tool.pylint.MASTER`.